### PR TITLE
Updated the README.md

### PR DESCRIPTION
**Pull request by PBOwner**
Updated the README.md to have links directly to the docker compose file (modify the links in the (<>) to where it would be for yours) and the .env,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
...
+## Env File:
+You can grav the variables from [here](<https://github.com/PBOwner/xpd/blob/prod/.env.example>)
+
+Make sure you replace `<token>` and both `<db_pass>` with your own bot token and database password for postgres. You do **NOT** need to configure postgres, just set those variables up and it will configure for you.
+
+## Finally, start the docker with:
+```bash
+docker compose up
+```
+If there are any errors, ping rosilla07 or valkyrie_pilot on the official discord [here](<https://discord.gg/sGX2EhceNc>)
```
Comment (randomairborne) [inline README.md:88]: I have a link for this already: [https://valk.sh/discord](https://valk.sh/discord). Also, I'm not sure how I feel about listing you as a potential help source. I'm not necessarily adverse to it, but it does seem like something worth discussing

File: README.md [hunk 2]
```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
...
+
+## Invite the bot with:
+
+```bash https://discord.com/oauth2/authorize?client_id=<yourclientid>&permissions=8&scope=bot+applications.commands```
```
Comment (randomairborne) [inline README.md:92]: this probably shouldn't be bash syntax highlighted. Also, experienced does *not* need administrator.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+services:
+  xpd-gateway:
+    image: ghcr.io/randomairborne/xpd-gateway:latest
+    restart: unless-stopped
+    networks:
+      - xpd-network
+    env_file:
+      - .env
+    environment:
+      - REDIS_URL=redis://redis:6379
```
Comment (randomairborne) [inline docker-compose.yml:14]: Do we maybe want to set the postgres URL automatically based on the POSTGRES_PASSWORD? We have all the other information.

File: README.md [hunk 4]
```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
 
 And you're done! This mini-tutorial doesn't support the leaderboard, though. You also may wish to set up a simple
 docker compose file. [Join the discord](https://valk.sh/discord) to learn more!
+
+
+# Using Docker Compose
+You can grab the file from [here](<https://github.com/PBOwner/xpd/blob/prod/docker-compose.yml>)
```
Comment (randomairborne) [inline README.md:77]: I think we need to fix the links here. IIRC you can directly link files with `./docker-compose.yml`, which would make it nice and repo-agnostic.

File: .env.example [hunk 5]
```diff
@@ -1,7 +1,6 @@
-DISCORD_TOKEN=
-DISCORD_PUBKEY=
-DATABASE_URL=
-REDIS_URL=
-ROOT_URL=
-CONTROL_GUILD=
-OWNERS=
\ No newline at end of file
+POSTGRES_PASSWORD=<db_pass>
+DATABASE_URL=postgres://xpd:<db_pass>@postgres:5432/xpd
```
Comment (randomairborne) [inline .env.example:2]: I think we might be able to automate the generation of this. see below.

**Review by randomairborne — CHANGES_REQUESTED** [review]

**Comment by randomairborne** [comment]
fyi, you can push code to your existing repo and it will update this PR- you don't have to make a new one if you don't want

File: README.md (hunk 4 above)
Comment (PBOwner) [reply README.md]: this is actually, for this instance, [here](/docker-compose.yml) for relative rather than absolute.

File: .env.example [hunk 6]
```diff
@@ -1,7 +1,6 @@
-DISCORD_TOKEN=
-DISCORD_PUBKEY=
-DATABASE_URL=
-REDIS_URL=
-ROOT_URL=
-CONTROL_GUILD=
-OWNERS=
\ No newline at end of file
+POSTGRES_PASSWORD=<db_pass>
+DATABASE_URL=postgres://xpd:$(POSTGRES_PASSWORD)@postgres:5432/xpd
```
Comment (randomairborne) [inline .env.example:2]: the env won't fill itself, you have to do this in the compose file

File: README.md [hunk 7]
```diff
@@ -71,3 +71,24 @@ docker run --env-file .env --add-host=host.docker.internal:host-gateway --detach
...
+## Env File:
+You can grav the variables from [here](/.env.example)
+
+Make sure you replace `<token>` and both `<db_pass>` with your own bot token and database password for postgres. You do **NOT** need to configure postgres, just set those variables up and it will configure for you.
+
+## Finally, start the docker with:
+```bash
+docker compose up
+```
+If there are any errors, ping valkyrie_pilot on the official discord [here](<https://valk.sh/discord>)
```
Comment (randomairborne) [inline README.md:88]: i don't think you need the angle brackets here?

File: docker-compose.yml (hunk 3 above)
Comment (randomairborne) [inline docker-compose.yml:14]: add a postgres url here, probably

**Review by randomairborne — CHANGES_REQUESTED** [review]

File: .env.example (hunk 6 above)
Comment (PBOwner) [reply .env.example:2]: I tested it locally and it worked-

File: README.md (hunk 7 above)
Comment (PBOwner) [reply README.md:88]: The angle brackets, while not necessary, help *some* with netness

File: docker-compose.yml (hunk 3 above)
Comment (PBOwner) [reply docker-compose.yml:14]: I can try this. lemme test locally before i update here

Comment (PBOwner) [reply docker-compose.yml:14]: can confirm that it doesn't work here.

File: .env.example (hunk 6 above)
Comment (randomairborne) [reply .env.example:2]: *what*
if so, that's news to me. i just don't see a good reason in keeping this in the .env, when none of the content is actually relevant to the user

File: README.md (hunk 7 above)
Comment (randomairborne) [reply README.md:88]: fair enough

File: docker-compose.yml (hunk 3 above)
Comment (randomairborne) [reply docker-compose.yml:14]: why not?

Comment (PBOwner) [reply docker-compose.yml:14]: doesn't read the user or the password. issue i was having before moving it to the .env
i am testing another location.

**Pull request merged**